tldw_Server_API/app/core/DB_Management/DB_Backups.py

```
@@ -76,7 +154,9 @@ def _validate_backup_name(backup_name: str, allowed_exts: tuple[str, ...]) -> Op
 
 def init_backup_directory(backup_base_dir: str, db_name: str) -> str:
     """Initialize backup directory for a specific database."""
-    backup_dir = os.path.join(backup_base_dir, db_name)
+    backup_dir = _safe_join(backup_base_dir, db_name) or ""
+    if not backup_dir:
+        raise ValueError("Invalid backup directory")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)